src/components/oscal-utils/OSCALComponentResolver.js

```diff
@@ -1,5 +1,16 @@
 import OSCALResolveProfileOrCatalogUrlControls from "./OSCALProfileResolver";
 
+function processesToRun(componentDefinition) {
+  if (!componentDefinition.components.forEach){ 
+    return 0;
+  }
```

Comment:
```suggestion
  if (!componentDefinition.components.forEach){ 
    return 0;
  }
```

This doesn't make any sense? `forEach` is part of the `Array.prototype`? And it's a function -- it's not even being invoked here? If you apply my other suggestion a `?.reduce` followed by `?? 0` might accomplish what you want here?

Reply:
This was added because I thought there were cases where `components` was an object, not an array.
Turns out the actual viewer resolves this as an array, but the test data passes in an object.
I believe the test data is incorrect and should be changed to pass in an array, but someone can correct me if that's wrong.

Reply:
Looking at [the specification outline](https://pages.nist.gov/OSCAL/reference/latest/complete/json-outline/), `components` must be an array of 1 or more elements (if `components` is present`). If the tests are passing in an object, the tests are not compliant with the specification.